style.css

```diff
@@ -0,0 +1,121 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Montserrat", sans-serif;
+  margin: 0;
+  padding: 0;
+  opacity: 0.70;
+}
+
+.first {
+  background-image: url("geio-tischler-GW0x6-5Fp2I-unsplash.jpg");
+  height: 100%;
+  width: 100%;
+  min-height: 700px;
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;

```

Comment:
why we used with `background-size:cover`

Reply:
To cover whole div space by image